src/components/webview/sourceControl.ts

```diff
@@ -20,6 +20,15 @@ export class SourceControlService {
 		private readonly __messageBus: MessageBus,
 	) {}
 
+	// @TODO
+	getBaseBranchName() {
+		return 'MainBranchName';
+	}
+
+	async createPR() {
```

Comment:
that's just a PR for the UI right?

Reply:
Yes, this PR is just UI, so methods in services are empty